static/scripts/topicEdit.js

```diff
@@ -299,6 +299,7 @@ class TopicBlockList extends React.Component {
      * Render the list items.
      */
     render() {
+        const neXboardEnabled = ($contentBlocksContainer.data('nexboardenabled') === true);
```

Comment:
has to be "true"!? because $contentBlocksContainer.data('nexboardenabled') is a string

Reply:
https://api.jquery.com/data/#data2